index.js

```diff
@@ -127,25 +145,22 @@ const drawCommand = (message) => {
   });
 };
 
-/*
-const inventoryCommand = () => {
-
-if user draws a card
-store name of card to firebase under the key cardName
-when the same user draws a second card, store the name of the second card under the previous draw within cardName
-continue to do this to all additional cards
-
-user{
-  userID{
-    lastDrawTime: xxxx
-    username: ""
-    cardName: "Blue Eyes White Dragon", "Red Eyes Black Dragon"
-  }
-}
+const inventoryCommand = (message) => {
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    const userIdentification = message.author.id;
+    const userInventory = userData[userIdentification].inventory;
+    let cardData = Object.values(userInventory);
+    let inventoryList = "";
```

Comment:
Like I mentioned before, `let` is a code smell. Here we thought it was okay because we are legitimately re-assigning this variable, but it's possible to do what we want without "mutating" (i.e. changing, re-assigning) this variable. In general you want to avoid or reduce mutation (we'll go into why later).

So let's use an array, like you had before :). in that array we can store every line we want to output, so the values in the variable ends up looking like this:

```js
[
  "•  Blue Eyes\n",
  "•  Red Eyes\n",
  "•  Green Eyes\n",
]
```

to populate the array, you can use `arrayName.push` as usual.

When you want to convert the array to string, a handy function is `arrayName.join()`. Check that out on MDN